I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/rmcountry.rpy", line 164, in script
    show Rix Normal at rightm with easeinright
Exception: Image 'Rix' does not accept attributes 'Normal'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/rmcountry.rpy", line 164, in script
    show Rix Normal at rightm with easeinright
  File "C:\Users\marakulina-u\Desktop\renpy-8.2.1-sdk\renpy\ast.py", line 1113, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\marakulina-u\Desktop\renpy-8.2.1-sdk\renpy\ast.py", line 1076, in show_imspec
    renpy.config.show(name,
  File "C:\Users\marakulina-u\Desktop\renpy-8.2.1-sdk\renpy\exports.py", line 739, in show
    if not base.find_target() and renpy.config.missing_show:
  File "C:\Users\marakulina-u\Desktop\renpy-8.2.1-sdk\renpy\display\image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "C:\Users\marakulina-u\Desktop\renpy-8.2.1-sdk\renpy\display\displayable.py", line 246, in _duplicate
    args.extraneous()
  File "C:\Users\marakulina-u\Desktop\renpy-8.2.1-sdk\renpy\display\displayable.py", line 109, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'Rix' does not accept attributes 'Normal'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.1.24030407
Markiz 1.0
Sun Apr 14 21:00:38 2024
